contracts/sol6/Dao/IKyberStaking.sol

```diff
@@ -18,7 +18,7 @@ interface IKyberStaking is IEpochUtils {
         returns (
             uint256 _stake,
```

Comment:
Is there a reason why the return params for this function have underscore prefix? Otherwise, seems to make more sense to align it with the other functions.